### DISCUSSION TITLE
Skippint tasks, `@task` decorator parameters and `soft_fail`

### DISCUSSION THREAD

**Discussion by rodigu** [apache/airflow#45034] [Q&A]
Where can I find the documentation for all available `@task` decorator parameters?

I want to have a task that gets skipped instead of failing when an error is thrown.
`soft_fail` does not seem to be a valid parameter to the `@task` decorator, seeing that I get an error when I try it.

**Comment by raphaelauv** [comment]
soft_fail is only for sensor
